package.json

```diff
@@ -39,15 +46,21 @@
     "nps-utils": "^1.2.0",
     "opt-cli": "^1.5.1",
     "prettier-eslint-cli": "^4.0.2",
-    "require-from-string": "^1.2.1",
+    "require-from-string": "1.2.1",
```

Comment:
Looks like this is still in `devDependencies`. I'd also prefer it to include the `^` :)